index.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
 function transformJs (src) {
   return buble.transform(src, {
     transforms: {
+      asyncAwait: false,
```

Comment:
I don't think this is a good idea because it will make people think they can use async await in their code which they cannot yet

Reply:
Yeah, I was thinking that. It is necessary for jest@27 and makes the argument for pinning to v26 stronger